### PR TITLE
Added cntrl+f11 shortcut for fullscreen

### LINKED ISSUE #122: Keyboard shortcut for full screen mode toggle

### PR DESCRIPTION
**Pull request by siddhant1**
Fixes #122 , will add to keyboard shortcuts roo

### CODE REVIEW

**Comment by domoritz** [comment]
Isn't that interfering with the browsers fullscreens shortcut?

**Comment by siddhant1** [comment]
That's why I added cntrl+f11 , it doesn't interfere

**Comment by siddhant1** [comment]
@domoritz can you check if this is working on mac?

File: src/components/renderer/renderer.tsx [hunk 1]
```diff
@@ -113,6 +113,18 @@ class Editor extends React.Component<Props, State> {
   public componentDidMount() {
     this.initView(this.props);
     this.renderVega(this.props);
+    // Add Event Listner to f11 key
```
Comment (domoritz) [inline src/components/renderer/renderer.tsx]: listener

Comment (siddhant1) [reply src/components/renderer/renderer.tsx]: :p Doing

**Comment by domoritz** [comment]
Works. I think it would be good to support CMD + F11 on mac as well.

**Comment by siddhant1** [comment]
> Works. I think it would be good to support CMD + F11 on mac as well.

I will look into it , What is the current behaviour?

**Comment by siddhant1** [comment]
Have you tested it @domoritz ? Tell me if any changes are required

**Comment by domoritz** [comment]
Cmd does not work yet.

**Comment by siddhant1** [comment]
Pushed , can you confirm now?

**Pull request merged**